### DISCUSSION TITLE
How would I extract a value for an entity

### DISCUSSION THREAD

**Discussion by acrigney** [explosion/spacy-llm#335] [General]
Guys I have an enourmous amount of medical reports to perfrom NER on. But I need not only to extract the entities but also values for those entities e.g if I need the Gleason score. The orginal text has "Gleason grade 4+3" but I need to give an attribute to the entity of 4+3. I am also creating knowledge graphs with neo4j and I would like to have the 4+3 as an attribute in the Gleason grade node.
How would you recommend that I do that? I guess I will need to modify the example parser?

**Comment by rmitsch** [comment]
Hi @acrigney, sounds like an interesting use case.

> The orginal text has "Gleason grade 4+3" but I need to give an attribute to the entity of 4+3.

I'm not sure if I follow. So the text literally says "Gleason grade 4+3"? What do you mean with "give an attribute"?

**Comment by rmitsch** [comment]
Converting this to a discussion.
